tests/test_masked_inference_wsi_dataset.py

```diff
@@ -0,0 +1,241 @@
+import os
+import unittest
+from unittest import skipUnless
+from urllib import request
+
+import numpy as np
+from numpy.testing import assert_array_equal
+from parameterized import parameterized
+
+from monai.apps.pathology.datasets import MaskedInferenceWSIDataset
+from monai.utils import optional_import
+from tests.utils import skip_if_quick
+
+_, has_cim = optional_import("cucim")
+_, has_osl = optional_import("openslide")
+
+FILE_URL = "http://openslide.cs.cmu.edu/download/openslide-testdata/Generic-TIFF/CMU-1.tiff"
+
+HEIGHT = 32914
+WIDTH = 46000
+
+mask = np.zeros((WIDTH // 2, HEIGHT // 2))
+mask[100, 100] = 1
+np.save("./tests/testing_data/mask1.npy", mask)
```

Comment:
please do this in a function otherwise it'll  start to write to disk when we import the test cases.

Reply:
Sure, will do.